lsp-mode.el

```diff
@@ -1536,33 +1536,33 @@ WORKSPACE is the workspace that contains the diagnostics."
       "Report new diagnostics to flymake."
       (funcall lsp--flymake-report-fn
                (-some->> (lsp-diagnostics)
-                 (gethash buffer-file-name)
```

Comment:
Bad formats here 😄

Reply:
They are correct but it is GH rendering the indentation in an odd way. Anyway, I removed the irrelevant changes.